fvm/evm/emulator/tracker.go

```diff
@@ -38,8 +38,10 @@ func (ct *CallTracker) CaptureRequiredGas(address types.Address, input []byte, o
 		ct.callsByAddress[address] = calls
 	}
 
+	var in []byte
+	copy(in, input)
```

Comment:
you could initiate the same length as the input, this should reduce one allocation.